<a href="https://colab.research.google.com/github/sinku29/Data-Science-Assignment/blob/main/Hypo_LabTAT.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

In [2]:
data=pd.read_csv("LabTAT.csv")
data.head()

Laboratory 1  Laboratory 2  Laboratory 3  Laboratory 4
0        185.35        165.53        176.70        166.13
1        170.49        185.91        198.45        160.79
2        192.77        194.92        201.23        185.18
3        177.33        183.00        199.61        176.42
4        193.41        169.57        204.63        152.60

In [5]:
print(data)
stats.f_oneway(data.iloc[:,0], data.iloc[:,1],data.iloc[:,2],data.iloc[:,3])

     Laboratory 1  Laboratory 2  Laboratory 3  Laboratory 4
0          185.35        165.53        176.70        166.13
1          170.49        185.91        198.45        160.79
2          192.77        194.92        201.23        185.18
3          177.33        183.00        199.61        176.42
4          193.41        169.57        204.63        152.60
..            ...           ...           ...           ...
115        178.49        170.66        193.80        172.68
116        176.08        183.98        215.25        177.64
117        202.48        174.54        203.99        170.27
118        182.40        197.18        194.52        150.87
119        182.09        215.17        221.49        162.21

[120 rows x 4 columns]


F_onewayResult(statistic=118.70421654401437, pvalue=2.1156708949992414e-57)


The given p-value =
2.1156708949992414
×
1
0
−
57

  is extremely small, indicating strong evidence against the null hypothesis.

At the 5% significance level (
α=0.05), if the p-value is less than
α, we reject the null hypothesis. Since
p<α (
2.1156708949992414×10
−57
 <0.05), we reject the null hypothesis.

Therefore, we can conclude that there is a significant difference in the average TAT among the different laboratories at the 5% significance level.